djangogirls/settings.py

```diff
@@ -48,6 +48,8 @@
     'storages',
     'markdown_deux',
 
+    'django_countries',
```

Comment:
+1 to using django_countries - we may be able to do some nice touches with the flags later too :+1: